src/WebApi/Application/UseCases/Question/GetInterviewQuestion/GetInterviewQuestionUseCase.cs

```diff
@@ -9,9 +9,9 @@ public class GetInterviewQuestionUseCase : IGetInterviewQuestionUseCase
 {
     private IOutputPort _outputPort;
 
-    private readonly IInterviewQuestionRepository _interviewQuestionRepository;
+    private readonly IQuestionRepository _interviewQuestionRepository;
```

Comment:
```suggestion
    private readonly IQuestionRepository _questionRepository;
```

Reply:
In doing this change the field name was changed but not it's references, those will also need to be renamed for the code to compile 😄